src/WebJobs.Extensions.DurableTask/OutOfProcMiddleware.cs

```diff
@@ -408,8 +401,8 @@ private static FailureDetails GetFailureDetails(Exception e)
             const string ExceptionDelimeter = "\nException: ";
             const string StackDelimeter = "\nStack: ";
 
-            ReadOnlySpan<char> messageSpan = rpcExceptionMessage;
-            int startException = messageSpan.IndexOf(ExceptionDelimeter);
+            ReadOnlySpan<char> messageSpan = rpcExceptionMessage.AsSpan();
+            int startException = messageSpan.IndexOf(ExceptionDelimeter.AsSpan());
```

Comment:
I notice a handful of these `.AsSpan()` and `ToArray()` casts have been introduced. What happened that leads to these being _now_ necessary?

Reply:
The change is that we've downgraded from .NET 6 to .NET Core 3.1 and have lost access to some of these convenient and/or high-performance enhancements.